src/discordtipbot/discordbot.cr

```diff
@@ -584,6 +586,14 @@ class DiscordBot
     reply(msg, "The bot is in #{guilds} Guilds and sees #{users} users (of which #{cached_users} users are guaranteed unique)")
   end
 
+  def exit(msg : Discord::Message)
+    id = msg.author.id
+    return reply(msg, "**ALARM**: This is an admin only command! You have been reported!") unless @config.admins.includes?(id)
+
+    @log.error("#{@config.coinname_short}: Shutdown requested by #{id}")
```

Comment:
Should this really be `error`? We completely intend to `exit`. I'd suggest `warn` instead.

Reply:
Oops...